presto-docs/src/main/sphinx/functions/lambda.rst

```diff
@@ -24,6 +24,13 @@ Most SQL expressions can be used in a lambda body, with a few exceptions:
 Lambda Functions
 ----------------
 
+.. function:: apply(T, function<T,U>) -> U
+
+    Apply the lambda function on the given variable::
```

Comment:
Does `... on the giving value` sounds better? @haozhun

Reply:
`value` sounds like it will only accept a constant value. Maybe we could use `to the first argument`?

Reply:
Let’s say “Apply the function to the given value”